### PR TITLE
Galileo plugin: base64-encode the bodies of req/res

### PR DESCRIPTION
**Pull request by SGrondin**

### CODE REVIEW

**Comment by thibaultCha** [comment]
LGTM

File: kong/plugins/log-serializers/alf.lua [hunk 1]
```diff
@@ -57,6 +59,7 @@ local _M = {}
 -- ngx_http_core_module when possible.
 -- Public for unit testing.
 function _M.serialize_entry(ngx)
+
```
Comment (thibaultCha) [inline kong/plugins/log-serializers/alf.lua]: whyyyyy

File: kong/tools/ngx_stub.lua [hunk 2]
```diff
@@ -26,6 +26,9 @@ _G.ngx = {
       return res_str, sub_made
     end
   },
+  encode_base64 = function(str)
+    return string.format("base64_%s", str)
```
Comment (thibaultCha) [inline kong/tools/ngx_stub.lua]: neat

File: kong/plugins/log-serializers/alf.lua (hunk 1 above)
Comment (SGrondin) [inline kong/plugins/log-serializers/alf.lua]: Added a line, then deleted it and forgot to remove the space. I'll amend the commit.

**Comment by thibaultCha** [comment]
This has to also be manually merged to `next`. Also, merging with fast-forwarding against `master` is preferred, which GitHub doesn't do.

**Pull request merged**